lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -3821,6 +3840,7 @@ class DecomposeComplexOpsPass
     addPatternIfTargetOpIsIllegal<DecomposeAtenWhereScalarOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenWhereScalarOtherOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenWhereScalarSelfOp>(patterns);
+    addPatternIfTargetOpIsIllegal<DecomposeAtenMaskedFillScalarOp>(patterns);
```

Comment:
You also need to add the op here: https://github.com/llvm/torch-mlir/blob/eb7bac7ca43ef76581f025fb1c0417308a29582a/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp#L335